### PR TITLE
An Introduction to Kotlin and it's basics

### PR DESCRIPTION
**Pull request by PrajjawalBanati**
I have started with kotlin language and want your contributors to look and help by making their contribution to it.

### CODE REVIEW

File: _data/projects/Kotlin_Language.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- kotlin-language
+- kotlin library
+- kotlin-extensions
+- kotlin-multiplatform
+upforgrabs:
+name: help wanted
+url: https://github.com/500060722/Kotlin_Language/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted"
+
+    
+
```
Comment (ritwik12) [inline _data/projects/Kotlin_Language.yml:16]: @500060722 Remove these lines please

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Kotlin_Language.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+site: https://github.com/500060722/Kotlin_Language
+tags:
+- kotlin
+- kotlin-language
+- kotlin library
+- kotlin-extensions
+- kotlin-multiplatform
+upforgrabs:
+name: help wanted
+url: https://github.com/500060722/Kotlin_Language/issues?q=is%3Aissue+is%3Aopen+label%3A"help+wanted"
```
Comment (ritwik12) [inline _data/projects/Kotlin_Language.yml:13]: @500060722 Please change this to https://github.com/500060722/Kotlin_Language/labels/help%20wanted and also correct the indentation of line 12 and 13. and that tag is `link` not `url`

File: _data/projects/Kotlin_Language.yml (hunk 1 above)
Comment (PrajjawalBanati) [reply _data/projects/Kotlin_Language.yml:16]: I have removed but still facing issues

Comment (PrajjawalBanati) [reply _data/projects/Kotlin_Language.yml:16]: I don't know why ?

**Comment by ritwik12** [comment]
@500060722 Not my business but why did you used your SAP id for username, You will leave college soon but your GitHub will remain with you

File: _data/projects/Kotlin_Language.yml (hunk 2 above)
Comment (ritwik12) [reply _data/projects/Kotlin_Language.yml:13]: @PrajjawalBanati ^^

**Pull request closed without merging**